app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationsFragment.kt

```diff
@@ -211,7 +211,9 @@ class NotificationsFragment :
             override fun onItemRangeInserted(positionStart: Int, itemCount: Int) {
                 if (positionStart == 0 && adapter.itemCount != itemCount) {
                     binding.recyclerView.post {
-                        binding.recyclerView.scrollBy(0, Utils.dpToPx(requireContext(), -30))
+                        if (getView() != null) {
```

Comment:
When does that happen (view being null here)?

Reply:
When the view gets destroyed before this code is executed. This can happen because of the `post` which makes it execute at the end of the message queue. Executing immediately is not possible because we need to wait for the RecyclerView to actually display the change before jumping.
You probably won't manage to reproduce this but with a lot of users some will definitely trigger it by chance.